org.mwc.debrief.lite/build.xml

```diff
@@ -101,6 +101,7 @@
 			<fileset dir="../org.geotools/libs" includes="*.jar"/>
 			<fileset dir="" includes="cmap_legacy.jar"/>
 			<fileset dir="../org.mwc.debrief.legacy" includes="debrief_legacy.jar"/>
+			<fileset dir="libs" includes="*.jar"/>

```

Comment:
@ayeshare - were we missing out on some libs in the build? Or, should this PR include another jar?

Reply:
We had not included the jars in the dist, it was already added to the build for compilation.
My change was to add the jars to the dist and also in the manifest classpath